stdlib/public/Concurrency/TaskLocal.swift

```diff
@@ -212,23 +218,27 @@ public final class TaskLocal<Value: Sendable>: Sendable, CustomStringConvertible
 // ==== ------------------------------------------------------------------------
 
 @available(SwiftStdlib 5.1, *)
+@usableFromInline
```

Comment:
Hmm. Same ABI issue here, this is adding a public symbol that wasn't there before.

Reply:
I'm confused about this one, isn't this effectively "call the swift_task_localValuePush" and the symbol already was there?

I'll change to `@alwaysEmitIntoClient` as you suggested but don't understand why, could you explain a bit?